### PR TITLE
Verificacion de la cuenta de este o no creada en firebase

### PR DESCRIPTION
**Pull request by Matarratas777**
Co-authored-by: Gaddiel Patiño <GaddPat0609@users.noreply.github.com>
Co-authored-by: EliasCode79 <EliasCode79@users.noreply.github.com>

# Description

check if the account is created in firebase and if it is shown in the console and if not, it is created in addition to being able to enter the session if the account is yours

Examples:
Imagen de WhatsApp 2023-01-26 a las 18 40 40

Imagen de WhatsApp 2023-01-26 a las 18 41 48

Imagen de WhatsApp 2023-01-26 a las 18 43 18

### CODE REVIEW

**Review by Zuack55 — APPROVED** [review]
Good job

**Review by IvanGodinez21 — APPROVED** [review]
LGTM 🤯

**Review by HenrikhWolf — APPROVED** [review]
LGTM 👍

File: src/views/pantallaRegis/pantallaRegis.js [hunk 1]
```diff
@@ -1,33 +1,110 @@
...
+      .then((userCredential) => {
+        // Signed in
+        const user = userCredential.user
+        console.log(user)
+        // ...
+      })
+      .catch((error) => {
+        const errorCode = error.code
+        const errorMessage = error.message
+        console.log(errorCode, errorMessage)
```
Comment (YilKanda) [inline src/views/pantallaRegis/pantallaRegis.js:41]: On GitHub, you should not leave debugging code.

Instead you can use an Alert.alert(), indicating some message like: "User not logged in"

File: src/views/pantallaRegis/pantallaRegis.js [hunk 2]
```diff
@@ -1,33 +1,110 @@
...
+        const token = credential.accessToken
+        // The signed-in user info.
+        const user = result.user
+        // ...
+      })
+      .catch((error) => {
+        // Handle Errors here.
+        const errorCode = error.code
+        const errorMessage = error.message
+        console.log(errorCode, errorMessage)
```
Comment (YilKanda) [inline src/views/pantallaRegis/pantallaRegis.js:61]: On GitHub, you should not leave debugging code.

Instead you can use an Alert.alert(), indicating some message like: "User not logged in"

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Pull request merged**